likes/views.py

```diff
@@ -1,3 +1,36 @@
-from django.shortcuts import render
+import json
+
+from django.http import JsonResponse
+from django.views import View
+
+from core.utils   import login_decorator
+from likes.models import Like
+from stays.models import Stay
+
+
+class LikeView(View):
+    @login_decorator
+    def post(self,request):
+        try:
+            data    = json.loads(request.body)
+            stay_id = data['stay_id']
+            user    = request.user
+            stay    = Stay.objects.get(id=stay_id)
+
+            like, is_created = Like.objects.get_or_create(
+                stay = stay,
+                user = user
+            )
+
+            result = True
+
+            if not is_created :
+                like.delete() 
+                result = False                
+
+            return JsonResponse({'message': 'SUCCESS', 'result': result}, status=200)
```

Comment:
```suggestion

            if not is_created :
                like.delete() 

            return JsonResponse({'message': 'SUCCESS', 'result': is_created}, status=200)
```
현진님~ like 여부를 client 쪽에서 확인하기 위해서 bool type의 result를 반환하시는거죠?

잘 알고 계시듯이 django의 `get_or_create()` orm은 `(object, bool)`과 같이 튜플 형식을 반환합니다. 
그리고 여기에는 오브젝트가 create 됐다면 True를 아니라면 Flase를 반환합니다.

이를 그대로 사용한다면 result라는 true, flase를 담는 변수의 필요없이 같은 의미의 데이터를 전달 할 수 있을 것 같네요!

Reply:
앗 그러네요! 굳이 어려운 방법으로 구현을 했었네요 감사합니다:D